### PR TITLE
AR recovery transaction backend

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/daemon.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def request(config: SimpleConfig, endpoint, args=(), timeout=60):
         auth = aiohttp.BasicAuth(login=rpc_user, password=rpc_password)
         loop = asyncio.get_event_loop()
         async def request_coroutine():
-            async with aiohttp.ClientSession(auth=auth, loop=loop) as session:
```
Comment (harzo) [inline electrum/daemon.py:103]: @str0g why this `loop` is removed?

File: electrum/three_keys/script.py [hunk 2]
```diff
@@ -15,8 +15,16 @@ def __init__(self, recovery_pubkey: str):
...
-        if not isinstance(public_keys, list) or len(public_keys) != 1:
-            raise ThreeKeysError(f"Wrong input type! Expected list not '{public_keys}'")
+        if not isinstance(public_keys, list) or len(public_keys) not in [1, 2]:
+            raise ThreeKeysError(f"Wrong input type! Expected 1 or 2 elements list not '{public_keys}'")
+
+        if len(public_keys) == 2:
+            if self.recovery_pubkey not in public_keys:
+                raise ThreeKeysError('No pubkey in input')
+            if self.recovery_pubkey == public_keys[0]:
+                del public_keys[0]
```
Comment (harzo) [inline electrum/three_keys/script.py:25]: @kj-crypto as discussed

Comment (kj-crypto) [reply electrum/three_keys/script.py]: done.

File: electrum/daemon.py (hunk 1 above)
Comment (harzo) [reply electrum/daemon.py:103]: https://github.com/aio-libs/aiohttp/issues/3331

**Pull request merged**